developersociety/commonslibrary
# Add sort order for organisation footer logos

# Sources
https://trello.com/c/Y2Fmnzxt/20-be-fe-make-partner-logos-configuable

# Setup instructions
`fab get_backup`
`fab get_media:uploads/directory`

# How to test
1. Go to http://127.0.0.1:8000/admin/directory/organisation/18/change/ and change **Logo sort order** to 10 go to http://127.0.0.1:8000/ and the `GetUp` logo should be shown first from the left
2. Repeat the for the founder http://127.0.0.1:8000/admin/directory/organisation/4/change/
